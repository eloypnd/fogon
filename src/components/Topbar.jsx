import React from 'react'
import { Avatar, Heading, Toolbar } from 'rebass'
import logo from '../logo.svg'

const Topbar = ({data}) => (
  <Toolbar>
    <Avatar bg='white' mr={2} size={24} src={logo} />
    <Heading fontSize={3}>fog√≥n</Heading>
  </Toolbar>
)

export default Topbar
