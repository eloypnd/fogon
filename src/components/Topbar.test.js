import React from 'react'
import renderer from 'react-test-renderer'
import Topbar from './Topbar'

it('Topbar renders correctly', () => {
  const tree = renderer
    .create(<Topbar />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
