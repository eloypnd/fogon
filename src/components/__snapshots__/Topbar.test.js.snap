// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Topbar renders correctly 1`] = `
<div
  className="sc-Rmtcm kOUKrJ sc-bwzfXH pjOvZ"
>
  <img
    className="sc-iAyFgw jLmdZJ"
    src="logo.svg"
  />
  <h2
    className="sc-VigVT cBIDWq"
  >
    fog√≥n
  </h2>
</div>
`;
